marathon-interface/src/main/scala/mesosphere/marathon/interface/auth/Identity.scala

```diff
@@ -0,0 +1,14 @@
+package mesosphere.marathon.interface.auth
+
+/**
+  * Base trait that represents an identity.
+  */
+trait Identity {
+
+  /**
+    * The string identifier of that identity.
+    * @return the unique identifier of that user.
+    */
+  def id: String
```

Comment:
Is this really needed?

I can imagine a few scenarios where no id is needed:

- Mocks in Tests
- Very rudimentary security where there's only one user with access to everything.
- Plugins that rely on class hierarchies instead of an id: AdminIdentity vs. UserIdentity